Framework/Algorithms/src/AddSampleLog.cpp

```diff
@@ -37,6 +37,15 @@ void AddSampleLog::init() {
                   boost::make_shared<StringListValidator>(propOptions),
                   "The type that the log data will be.");
   declareProperty("LogUnit", "", "The units of the log");
+
+  std::vector<std::string> typeOptions;
+  typeOptions.push_back("Int");
```

Comment:
Should have static const std::string for Int and Double. Can then use elsewhere in the code instead of writing "Int", "Double"